src/server/json_family.cc

```diff
@@ -627,9 +627,15 @@ OpResult<vector<OptString>> OpArrPop(const OpArgs& op_args, string_view key, str
     }
 
     auto it = std::next(val.array_range().begin(), removal_index);
-    ostringstream os;
-    os << pretty_print(*it);
-    vec.emplace_back(os.str());
+    string str;
```

Comment:
if you do not use stringstream anymore, then please remove the include as well.

Reply:
I've not added include to `sstring` in my previous commit, so no need to remove the include.